pkg/sos/sosServer.go

```diff
@@ -2,7 +2,7 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
```

Comment:
some of these names "stutter" -- I think sos/server.go is fine, I don't think you need sos/sosServer.go

Reply:
Fixed